src/Geopilot.Frontend/public/locale/en/common.json

```
@@ -48,7 +48,7 @@
   "fieldOptional": "Is optional",
   "fieldRequired": "Is required",
   "fileDropzoneErrorChooseFile": "Please choose a valid file",
-  "fileDropzoneErrorFileTooLarge": "The selected file is larger than 200MB",
+  "fileDropzoneErrorFileTooLarge": "The selected file is larger than 100MB",
```

REQ: Abstände überall beibehalten wie in dropzone.
```suggestion
  "fileDropzoneErrorFileTooLarge": "The selected file is larger than 100 MB",
```